NameGame/NameGame/State/Reducer/GameReducer.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  GameReducer.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/21/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import ReSwift
+import UIKit
+
+enum GameAction: Action {
+    case setImages(Loading<[UIImage]>)
+    case setStatus(GameState.Status)
+    case playerAnswersCorrectly
+    case goToNextQuestion(Question)
+    case gameTerminated
```

Comment:
Let's move this action to the `AppAction` enum and change it's name to `endGame`. This way our `GameReducer` is only responsible for mutations to the current game.